### ISSUE TITLE
No normative statement wrt. JWT encoding of `verifiableCredential` as base64url

### ISSUE THREAD

**Issue by msporny** [w3c/vc-data-model#832]
Example 31 in the specification currently presumes base64url-encoding when embedding VCs in JWT-encoded VPs:

https://w3c.github.io/vc-data-model/#example-31-jwt-payload-of-a-jwt-based-verifiable-presentation-non-normative

There is no normative specification language related to this particular transformation to/from base64url.

**Comment by David-Chadwick** [comment]
@msporny This is not needed because it is part of the JWT specification in RFC 7519. So any VC that is proofed using a JWT is automatically base64URL encoded. Perhaps all that is needed is a reference to RFC 7519. This can be an editorial addition to v1.1. Can you label this issue accordingly please

**Comment by msporny** [comment]
> Can you label this issue accordingly please.

This issue is about adding normative language to the specification to make it clear that `verifiableCredential`, when encoded in a JWT, MUST be transformed into a base64url encoded value. It is also about adding normative language to the specification that states that when transforming from a JWT, that the `verifiableCredential` value MUST be transformed from the base64url value back into the data model. The specification is not clear about that today, and given that these are normative changes, we can't mark it as editorial.

Let's discuss in the call today.

**Comment by David-Chadwick** [comment]
I dont think this is necessary. The specification can say that JWTs are encoded according to RFC 7519, which is already a normative reference in the data model specification.
